spec/features/user_can_edit_station_spec.rb

```diff
@@ -21,9 +21,9 @@
     fill_in('station[city]', :with => "New City")
     fill_in('station[dock_count]', :with => "38")
     fill_in('station[installation_date]', :with => "07/07/2016")
-    click_on('submit')
+    click_on('Submit')
```

Comment:
Can't comment above since it's not the most recent commit, but:

Line 2: If you're going to use Faker in multiple tests, recommend putting this require into your `spec_helper`. Then you won't need to require it in each test.
Line 5: Looks like you copied from another test (which is fine), but forgot to update the description.
Lines 16, 19, and 26: Test might be slightly less brittle if you save the Station you create to a variable, and then use it when referring to your paths as `"/stations/#{station.id}".